src/PROPOSAL/PROPOSAL/propagation_utility/Interaction.h

```diff
@@ -47,6 +48,7 @@ class Interaction {
 
     double MeanFreePath(double energy);
 
+    auto GetLowerLim() const noexcept { return lower_lim; }
```

Comment:
Calling this one `GetLowerLim` might be confusing since there already is a function with the same same in `CrossSectionVector` with a different meaning